modules/apps/feature-flag/feature-flag-test/src/testFunctional/tests/FeatureFlags.testcase

```diff
@@ -0,0 +1,85 @@
+@component-name = "portal-feature-flags"
+definition {
+
+	property custom.properties = "feature.flag.LPS-xxxxxx=false${line.separator}feature.flag.LPS-xxxxxx.status=beta${line.separator}feature.flag.LPS-xxxxxx.title=Terms and Conditions pages${line.separator}feature.flag.LPS-xxxxxx.description=Create your own terms and conditions pages${line.separator}feature.flag.LPS-yyyyyy=false${line.separator}feature.flag.LPS-yyyyyy.status=beta${line.separator}feature.flag.LPS-yyyyyy.dependencies=LPS-xxxxxx";
```

Comment:
feature.flag.LPS-yyyyyy.status change to feature.flag.LPS-yyyyyy.type, also feature.flag.LPS-xxxxxx.type

Reply:
Done